tests/rptest/services/chaos/workloads/list_offsets/service.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2024 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import requests
+import signal
+from enum import Enum
+import dataclasses
+
+from ducktape.services.service import Service
+from ducktape.cluster.remoteaccount import RemoteCommandError
+from ducktape.errors import TimeoutError
+
+from rptest.util import wait_until
+
+from ...types import NoProgressError
+
+
+class NodeState(str, Enum):
+    INITIALIZED = 'initialized'
+    STARTED = 'started'
+    STOPPED = 'stopped'
+
+
+@dataclasses.dataclass
+class WorkloadInfo:
+    succeeded_ops: int = 0
+    failed_ops: int = 0
+    timedout_ops: int = 0
+    is_active: bool = False
+
+
+class ListOffsetsWorkload(Service):
+    PERSISTENT_ROOT = "/var/lib/chaos_workloads/list_offsets"
+
+    def __init__(self, ctx, brokers_str, topic):
+        super().__init__(ctx, num_nodes=1)
+
+        self._remote_port = 8080
+        self._pids = dict()
```

Comment:
do we need this map at all ?
We can always query a node with `node.account.java_pids` this way we will be able to avoid errors where the map is out of sync with the actual processes running on nodes

Reply:
The relevant apis: 
```
node.account.alive(process_pid)
node.account.java_pids(cmd_name)
```

Reply:
Done